cltk/phonology/syllabify.py

```diff
@@ -59,8 +59,7 @@ def syllabify_SSP(self, word):
 
             >>> fricatives = ['f']
 
-            >>> s = Syllabifier(high_vowels=high_vowels, mid_vowels=mid_vowels, low_vowels=low_vowels, flaps=flaps,\\
-            nasals=nasals, fricatives = fricatives)
+            >>> s = Syllabifier(high_vowels=high_vowels, mid_vowels=mid_vowels, low_vowels=low_vowels, flaps=flaps, nasals=nasals, fricatives = fricatives)
```

Comment:
fricatives = fricatives -> fricatives=fricatives (just a question of code style)

Reply:
Ahhh you're right, it's just sloppy coding I'm afraid 🙄 Let me fix it.